### DISCUSSION TITLE
General error : The system cannot find the path specified.

### DISCUSSION THREAD

**Discussion by Justinchua-L** [spatie/laravel-backup#1542] [Q&A]
I have trying to use this package for my laravel project to backup the database . 

Database:mysql
LARAVEL Version: 8 

The things is do is 
1. run composer require spatie/laravel-backup
2. put  "Spatie\Backup\BackupServiceProvider::class," under config/app.php 
3. run php artisan vendor:publish --provider="Spatie\Backup\BackupServiceProvider"
4. update my config/database.php
its like this

 'mysql' => [
            'driver' => 'mysql',
            'url' => env('DATABASE_URL'),
            'host' => env('DB_HOST', '127.0.0.1'),
            'port' => env('DB_PORT', '3306'),
            'database' => env('DB_DATABASE', 'forge'),
            'username' => env('DB_USERNAME', 'forge'),
            'password' => env('DB_PASSWORD', ''),
            'unix_socket' => env('DB_SOCKET', ''),
            'charset' => 'utf8mb4',
            'collation' => 'utf8mb4_unicode_ci',
            'prefix' => '',
            'prefix_indexes' => true,
            'strict' => true,
            'engine' => null,
            'options' => extension_loaded('pdo_mysql') ? array_filter([
                PDO::MYSQL_ATTR_SSL_CA => env('MYSQL_ATTR_SSL_CA'),
            ]) : [],
            'dump' => [
                'dump_binary_path' => 'D:\xampp\mysql\bin/', // only the path, so without `mysqldump` or `pg_dump`
                'use_single_transaction',
                'timeout' => 60 * 5, // 5 minute timeout
                          
            ],
        ],

but i got the following erros here : Backup failed because: The dump process failed with exitcode 1 : General error : The system cannot find the path specified.

Can anyone give me a help, im been struggling with this for 3 days
![erro sql dump](https://user-images.githubusercontent.com/59947335/176647527-7ad42bcf-a251-4c15-9a0c-d5ae96800034.PNG)

**Comment by erikn69** [comment]
> put "Spatie\Backup\BackupServiceProvider::class," under config/app.php

It is innescessary, it is automatic

> 'dump_binary_path' => 'D:\xampp\mysql\bin/', 

put mysql bin on `environment` path variable, and delete that entrie

And the config that you wrote is different to the image, `C:\\` and `D:\\`

**Reply by Justinchua-L** [reply]
![environment path 2](https://user-images.githubusercontent.com/59947335/176730886-e46b0394-505f-4314-ac64-148fdf368dac.PNG)
Hi I have already add path to the environment variable , but its still not working ？ did i miss other things?

and im using the the path D:\xammp\mysql\bin
![environment path 1](https://user-images.githubusercontent.com/59947335/176730972-95c5b3e4-29fe-4c1d-accb-7a3024282564.PNG)
![environment path 2](https://user-images.githubusercontent.com/59947335/176730976-8286f8d4-7d84-4214-84a7-7204043b255d.PNG)
![sql dump d](https://user-images.githubusercontent.com/59947335/176730980-6f698fa1-672a-4cba-88fd-fddfd41389e7.PNG)

**Reply by erikn69** [reply]
> and delete that entrie

You don't follow that instruction

And maybe the path where the backup is saved doesnt exists, or the temp path doesnt exists

Also: [requirements](https://spatie.be/docs/laravel-backup/v8/requirements)
> It's not compatible with Windows

**Comment by Justinchua-L** [comment]
oh sorry , can you tell me which part should i delete ? this is my first time to use spatie  for backup . 


and regarding to the path , is there a way to know the correct path location ? like execure some file in cmd . thanks for the reply and your patient to answer my question.

**Reply by erikn69** [reply]
Read again and you find wich part must be deleted

**Reply by Justinchua-L** [reply]
I have try to delete the part needed to be deleted but just showing the same  result , anyways thanks for the help . i will still looking for other  solutions
